visualization/app/codeCharta/ui/codeMap/codeMap.actions.service.ts

```diff
@@ -60,7 +60,7 @@ export class CodeMapActionsService {
 		for (const edge of edges) {
 			edge.visible = EdgeVisibility.none
 
-			if (edge.attributes[edgeMetric] !== undefined) {
+			if (edge.attributes[edgeMetric] !== undefined && edgePreviewNodes !== null) {
```

Comment:
A set may never be null :-)

```suggestion
			if (edge.attributes[edgeMetric] !== undefined) {
```